### PR TITLE
Document how to display a child admin

### PR DESCRIPTION
**Pull request by core23**
I am targeting this branch, because this is a doc change for the latest stable.

Rebase of #3807 

## Subject

Improved getting architecture docs.

### CODE REVIEW

File: Resources/doc/reference/architecture.rst [hunk 1]
```diff
@@ -316,43 +316,79 @@ template files, administration panel title and logo.
...
+
+To display the ``VideoAdmin`` extend the menu in your ``PlaylistAdmin`` class:
+
+.. code-block:: php
+
+    <?php
+
+    namespace AppBundle\Admin;
+
+    class PlaylistAdmin extends AbstractAdmin
```
Comment (OskarStark) [inline Resources/doc/reference/architecture.rst]: would be great to have the used namespaces included in the example, AbstractAdmin, MenuItemInterface, AdminInterface

File: Resources/doc/reference/architecture.rst [hunk 2]
```diff
@@ -316,43 +316,83 @@ template files, administration panel title and logo.
...
+
+    use Knp\Menu\ItemInterface as MenuItemInterface;
+    use Sonata\AdminBundle\Admin\AbstractAdmin;
+    use Sonata\AdminBundle\Admin\AdminInterface;
+
+    class PlaylistAdmin extends AbstractAdmin
+    {
+        // ...
+
+        protected function configureTabMenu(MenuItemInterface $menu, $action, AdminInterface $childAdmin = null)
```
Comment (jordisala1991) [inline Resources/doc/reference/architecture.rst]: `configureTabMenu` is deprecated, use `configureSideMenu` instead

Comment (core23) [reply Resources/doc/reference/architecture.rst]: Good catch!

**Comment by greg0ire** [comment]
Thanks @core23 !

**Pull request merged**